README.md

```diff
@@ -8,7 +8,7 @@ infrastructure.
 Setup
 -----
 
-After cloning the repository, do the following (might require **sudo** in some distributions, and `pip` > 21.0.1):
```

Comment:
`make install` does not work in MacOS because of the above reasons, but `uninstall` still requires `sudo` so far. Probably won't after moving the install dir, but need to check before remove that).

Reply:
Has this been fixed since @robioson?

Reply:
In that you never have to run `sudo make`, yes